Common/cpp/Tools/RuntimeDecorator.cpp

```diff
@@ -4,22 +4,16 @@
 #include <memory>
 #include <unordered_map>
 #include <utility>
+#include "JSISerializer.h"
 #include "JsiUtils.h"
 #include "ReanimatedHiddenHeaders.h"
 
 namespace reanimated {
 
 static const std::function<void(jsi::Runtime &, jsi::Value const &)> logValue =
     [](jsi::Runtime &rt, jsi::Value const &value) {
-      if (value.isString()) {
-        Logger::log(value.getString(rt).utf8(rt).c_str());
-      } else if (value.isNumber()) {
-        Logger::log(value.getNumber());
-      } else if (value.isUndefined()) {
-        Logger::log("undefined");
-      } else {
-        Logger::log("unsupported value type");
-      }
+      auto serializedValue = stringifyJSIValue(rt, value);
+      Logger::log(serializedValue);
```

Comment:
Sorry for another suggestion on this line but I can't resist 😄 
```suggestion
      Logger::log(stringifyJSIValue(rt, value));
```